mycli/main.py

```diff
@@ -747,6 +749,13 @@ def format_output(self, title, cur, headers, status, expanded=False,
 
         return output
 
+    def get_reserved_space(self):
+        """Get the number of lines to reserve for the completion menu."""
+        reserved_space_ratio = .2
+        max_reserved_space = 8
+        _, height = click.get_terminal_size()
+        return min(int(height * reserved_space_ratio), max_reserved_space)
```

Comment:
Is 0.2 the right ratio? It requires you to have 10 lines before you get a completion menu that is 2 lines tall. 

Also what happens if the height comes back as 4 lines or lesser? Does it not show any completion menu at all?